conda_mirror/conda_mirror.py

```diff
@@ -456,7 +521,8 @@ def main(upstream_channel, target_directory, temp_directory, platform,
     # 1. validate local repo
     # validating all packages is taking many hours.
     # _validate_packages(repodata=repodata,
-    #                    package_directory=local_directory)
+    #                    package_directory=local_directory,
+    #                    num_threads=num_threads)
```

Comment:
Shame on me for leaving commented code in this project. 

👍  to you for updating commented code with the new API 😀